Sources/Drill/Invocation.swift

```diff
@@ -84,8 +85,7 @@ public struct Invocation {
       let url = URL(fileURLWithPath: path)
 
       func run<PassTy: PassProtocol>(_ pass: PassTy) -> PassTy.Output?
-        where PassTy.Input == URL
-      {
```

Comment:
Can't figure out how to convince it to let us do this.  For the sake of consistency, let's just go back to what you did before.